engine/src/main/java/org/terasology/engine/rendering/nui/layers/mainMenu/moduleDetailsScreen/ModuleDetailsScreen.java

```diff
@@ -286,9 +286,9 @@ public String get() {
         });
     }
 
-    private boolean validateModuleDependencies(Name moduleName) {
+    private boolean validateModuleDependencies(Name module) {
```

Comment:
I do think that nearly all of the renames should be reverted. These renames kind of work and are not bad per se, but it should not be necessary.

Part of my reasoning is that I do think that having a class member `UILabel moduleName` is fine and can happily co-exist with a parameter `Name moduleName`.
I'm not a fan of repeating the type in the name, e.g., naming the class member `moduleNameLabel` instead - there is a type system, why should I put it in the name...

Reply:
I reverted the HiddenFields related commit.